services/management/annotation.go

```diff
@@ -0,0 +1,83 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package management
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/percona/pmm/api/managementpb"
+	"gopkg.in/reform.v1"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+// AnnotationService Annotation Service.
+type AnnotationService struct {
+	db            *reform.DB
+	grafanaClient grafanaClient
+}
+
+// NewAnnotationService create new Annotation Service
+func NewAnnotationService(db *reform.DB, grafanaClient grafanaClient) *AnnotationService {
+	return &AnnotationService{
+		db:            db,
+		grafanaClient: grafanaClient,
+	}
+}
+
+// AddAnnotation create annotation in grafana
+func (as *AnnotationService) AddAnnotation(ctx context.Context, authorizationHeaders []string, req *managementpb.AddAnnotationRequest) (*managementpb.AddAnnotationResponse, error) {
+	tags := req.Tags
+	if len(req.ServiceNames) == 0 && req.NodeName == "" {
+		tags = append([]string{"pmm_annotation"}, tags...)
+	}
+	postfix := []string{}
```

Comment:
```go
var postfix []string
```

https://dave.cheney.net/practical-go/presentations/gophercon-singapore-2019.html#_use_a_consistent_declaration_style (but please read the whole piece)